android-sdk/src/main/java/com/mobilecoin/lib/MobileCoinClient.java

```diff
@@ -252,27 +280,77 @@ public AccountSnapshot getAccountSnapshot(UnsignedLong blockIndex) throws Networ
         return new AccountSnapshot(this, txOuts, finalBlockIndex);
     }
 
+    @Deprecated
     @Override
     @NonNull
-    public Balance getBalance() throws InvalidFogResponse, NetworkException, AttestationException {
+    public Balance getBalance() throws AttestationException, InvalidFogResponse, NetworkException {
+        try {
+            return getBalance(KnownTokenId.MOB.getId());
+        } catch (FogSyncException e) {
+            throw new NetworkException(NetworkResult.INTERNAL, e);
+        }
+    }
+
+    @Override
+    @NonNull
+    public Balance getBalance(UnsignedLong tokenId) throws AttestationException, InvalidFogResponse, NetworkException, FogSyncException {
         Logger.i(TAG, "GetBalance call");
-        return getAccountSnapshot().getBalance();
+        return getAccountSnapshot().getBalance(tokenId);
+    }
+
+    @Override
+    @NonNull
+    public Map<UnsignedLong, Balance> getBalances() throws AttestationException, InvalidFogResponse, NetworkException, FogSyncException {
```

Comment:
hard to deduce what `UnsignedLong` key is used for, can this be an enum/class so it's clear what token types are supported explicitly.